libraries/analysis/package.json

```diff
@@ -40,6 +40,8 @@
     "format:check": "prettier --config ../../.prettierrc.json --ignore-path ../../.prettierignore --check .",
     "lint": "eslint --config ../../.eslintrc.json --ignore-path ../../.eslintignore .",
     "lint:fix": "eslint --config ../../.eslintrc.json --ignore-path ../../.eslintignore . --fix",
+    "postinstall": "cp ../../NOTICE.header.ts ./",
```

Comment:
So you bassically ignored the header files such that they are not committed but after installation you do copy them to the separate folders, to avoid duplication right?

Reply:
Yes, exactly this. It was a bit frustrating to edit the same file in ~25 different places.

Reply:
The same happens for the LICENSE (which was currently even missing) and NOTICE files

Reply:
Awesome!